src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -236,6 +237,20 @@ public class GenerateProtoTask extends DefaultTask {
     public String getName() {
       return name
     }
+
+    public setOutputDir(String outputDir) {
```

Comment:
Maybe `setOutputSubDir()`, which reminds the user that this is always relative to the base output dir.
Also please add docs for this method.